src/test/java/duke/task/TodoTest.java

```diff
@@ -0,0 +1,21 @@
+package duke.task;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
You should move this static import to the top, and use a line to separate it from the normal import.